build.clj

```diff
@@ -0,0 +1,76 @@
+(ns build
+  (:refer-clojure :exclude [test compile])
+  (:require [clojure.tools.build.api :as b]
+            [borkdude.gh-release-artifact :as gh]
+            [org.corfield.build :as bb])
```

Comment:
this is deprecated unfortunately, just copy the build.clj from datahike-s3 or others

Reply:
Ah, then the datalog-parser probably needs an update as well as this is where I copied the stuff from

Reply:
Done.